core/color.cpp

```diff
@@ -398,12 +398,17 @@ Color Color::named(const String &p_name) {
 	}
 	String name = p_name;
 	// Normalize name
-	name = name.replace(" ", "");
-	name = name.replace("-", "");
-	name = name.replace("_", "");
+	// name = name.replace(" ", "");
+	// name = name.replace("-", "");
+	// name = name.replace("_", "");
+	// name = name.replace("'", "");
+	// name = name.replace(".", "");
+	// name = name.to_lower();
```

Comment:
Remove commented-out code, people wanting to see the history can always check it through `git`.

Reply:
Of course.